recognition/advanced_tree_search.py

```diff
@@ -855,19 +864,20 @@ def cleanup_before_run(self, cmd, retry, *args):
         util.backup_if_exists("build_global_cache.log")
 
     @classmethod
-    def create_config(cls, crp, extra_config, extra_post_config, **kwargs):
-        config, post_config = rasr.build_config_from_mapping(
-            crp,
-            {
-                "lexicon": "speech-recognizer.model-combination.lexicon",
-                "acoustic_model": "speech-recognizer.model-combination.acoustic-model",
-                "language_model": "speech-recognizer.model-combination.lm",
-                "recognizer": "speech-recognizer.recognizer",
-            },
-        )
+    def create_config(cls, crp, search_type, recognizer_type, extra_config, extra_post_config, **kwargs):
+        mapping_dict = {
+            "lexicon": "speech-recognizer.model-combination.lexicon",
+            "acoustic_model": "speech-recognizer.model-combination.acoustic-model",
+            "language_model": "speech-recognizer.model-combination.lm",
+            "recognizer": "speech-recognizer.recognizer",
+            "label_scorer": "speech-recognizer.model-combination.label-scorer",
```

Comment:
will unconditionally adding `"label_scorer"` here not make it be included in every config, breaking all our hashes in the process?

even better: the pipeline crashes with 
```
AttributeError: label_scorer_config
```
here if label_scorer is not present in the crp.


Maybe a better solution would be 
```
mapping_dict = {...}
if recognizer_type == "recognizer-v2":
  mapping_dict["label_scorer"] =  "speech-recognizer.model-combination.label-scorer"
```